ansible/roles/postldap/defaults/main.yml

```diff
@@ -416,7 +416,7 @@ postldap__postfix__dependent_lookup_tables:
                    )'
     result_attribute: ''
     special_result_attribute: 'member'
-    leaf_result_attribute: 'uid'
+    leaf_result_attribute: 'uid, mail'
```

Comment:
This to allow the use of the email as login. Therefore, I propose to use the `mail` attribute since it is authentication. However, It might be better to use `mailAddress` and `mailAlternateAddress`.

Reply:
Agreed, I think that `mailAddress` and `mailAlternateAddress` might be good here. I'm thinking about adding a new attribute in the schema, `mailReceiveAddress`, for read-only or historical addresses of a given user. These should not be allowed to be used as logins, so having only the two mentioned attributes would work well with that.

Conversely, `saslauthd` role might need to be updated to allow SMTP AUTH using these attributes instead of `inetOrgPerson`, but that probably be done in a separate PR.